### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jonthegeek**
Lots of videos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,22 +18,20 @@ This repo contains 3 main folders:
 
 *Cohort 2 [Americas]: (started 2020-07-30) - Thursdays, 7:30pm EDT/EST*
 
-- 2020-11-12 Chapter 15: S4
-- 2020-11-19 Chapter 16: Trade-offs & OOP Recap
-- 2020-12-03 Chapter 17: Big picture (of Metaprogramming)
+- 2020-11-19 Chapter 17: Big picture (of Metaprogramming)
```
Comment (tanho63) [inline README.md:21]: I think they’re past this now

File: README.md [hunk 2]
```diff
@@ -176,15 +175,17 @@ Chapter 14 - R6
 Chapter 15 - S4
 
 - Cohort 1: Pavitra Chakravarty ([slides](https://r4ds.github.io/bookclub-Advanced_R/Presentations/Week14/Cohort1/Chapter15.html#1) | [video](https://www.youtube.com/watch?v=a1jzpWiksyA&feature=youtu.be))
+- Cohort 2: Erick Knackstedt (SLIDES | [video](https://youtu.be/bzo37PHCM1I))
```
Comment (tanho63) [inline README.md:178]: Slides are openPR in #234 as well, integrate here?

**Review by tanho63 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tanho63) [reply README.md:178]: I think there’s a link to chapter four as well

Comment (jonthegeek) [reply README.md:178]: Yup, just came here to merge that PR in, sorry about that...

Comment (jonthegeek) [reply README.md]: It was chapter 5. Merged that PR into this one and resolved conflicts.

**Review by tanho63 — APPROVED** [review]

**Pull request merged**